WooCommerce/Classes/ViewRelated/InAppFeedback/InAppFeedbackCardViewController.swift

```diff
@@ -7,7 +7,8 @@ import StoreKit
 protocol SKStoreReviewControllerProtocol {
     /// Displays the in app app store review alert.
     ///
-    static func requestReview()
+    @available(iOS 10.3, *) static func requestReview()
```

Comment:
Now that I think about it, does it make sense to support `requestReview()` at all? The current app target is 14.0 so I guess we can just go ahead with `requestReview(in:)` ?

Reply:
Yeah, I think we should switch to the iOS 14 method entirely since that's our minimum target.

Reply:
Thanks for confirming, modified on 16036f0389207b2639c34b01b558f247d4da46dd :)